docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,106 @@
+    - TV UI Controls and CircularUI: New UI controls for Smart TVs and Smart Watches.
+    - XAML Hot Reload: Boosting developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device various offloading services
+
+As the complexity of applications in the web environment increases, a method of maximizing computational power is needed.
+
+Tizen 6.0 provides 'multi-device offloading service' technology using multiple devices.
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen java script engine for the product having Hardware resource restriction.
+It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine.
+It supports ES6 full specification and supports up to a large part of ES 10.
+
+- **Lightweight JavaScript engine (ESCARGOT)** in https://github.com/Samsung/excargot
+
+  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
+    - Binary size : 13.7 %
+    - Average RSS : 35.1 %
+
+  - Features
+    - Support ES6 Full feature and partial support of ES10
+
+## Machine Learning framework
+
+NNSteamer is an on-device AI framework that makes it easy to configure complex artificial neural networks in the application by making artificial neural network configurations with Gstreamer's plugin. 	 
+
+In Tizen 6.0, NN Runtime support such as control-flow and dynamic tensor has been added to support voice model acceleration. In addition, by providing the NN Compiler front-end for NN Runtime, it can convert well-known models such as Tensorflow and TensorFlow-lite to be used on CE devices with insufficient resources (on NNRuntime). 	 
+
+- NNStreamer 	 
+ - Neural Network Pipeline. 	 
+   - .NET and C APIs supported. 	 
+   - Various accelerating hardware & frameworks supported. 	 
+   - Implement AI app more easily. 	 
+   - Execute AI app more efficiently. 	 
+
+- NNTrainer 	 
+ - On-Device Training. 	 
+   - Transfer learning, meta learning, and reinforcement learning. 	 
+   - APIs to construct and update (learn!) neural network layers.
+
+- NN Runtime 	 
+ - Tizen’s default Neural Network inference framework (since v5.0).
+ - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
+ - Useful features for speech model support (in v6.0). 	 
+   - Control-flow, Dynamic Tensor, Shape Inference, etc. 	 
+
+- NN Compiler in Tizen SDK 	 
+ - Support well known ML frameworks, and model formats. 	 
+   - Tensorflow, Tensorflow-lite (preferential), Caffe, ONNX (experimental). 	 
+   - Easy scalability through Common IR and NN Package.
+
+
+   <figure align='center'>
+     <img src='media/6.0_1_NNRuntime.png'>
+     <figcaption >[ NN Runtime ]</figcaption>
+   </figure>
+
+   <figure align='center'>
+     <img src='media/6.0_1_NNStreamer.png'>
+     <figcaption >[ NNStreamer ]</figcaption>
+   </figure>
```

Comment:
We need to put the image & its caption in MD format. 
Like this:

    **Figure: NN Runtime**

    ![NN Runtime](./media/6.0_1_NNRuntime.png)


    **Figure: NN Streamer**

    ![NN Streamer](./media/6.0_1_NNStreamer.png)